app/styles/pages/bulletin-index.scss

```diff
@@ -35,7 +35,15 @@
     position: relative;
 
     section {
-      padding: 100px 0;
+      padding-bottom: 100px;
+      padding-top: 100px;
+    }
+
+    h1 {
+      font-size: 14px;
+      text-transform: uppercase;
```

Comment:
Properties should be ordered font-size, margin-bottom, text-align, text-transform